rotkehlchen/tests/unit/test_data_updates.py

```diff
@@ -553,3 +599,64 @@ def test_accounting_rules_updates(data_updater: RotkiDataUpdater) -> None:
             'counterparty': 'test_counterparty',
         },
     ]
+
+
+def test_location_asset_mappings_updates(
+        data_updater: RotkiDataUpdater,
+        globaldb: 'GlobalDBHandler',
+) -> None:
+    """Test that remote updates for location asset mappings work"""
+    # check state of the location asset mappings before updating
+    with globaldb.conn.read_ctx() as cursor:
+        assert cursor.execute('SELECT COUNT(*) FROM location_asset_mappings').fetchone()[0] == 1500
```

Comment:
This constant will change when we add more right? Define a variable named

```python
NUM_ASSETS_MAPPINS_V1_32: Final = 1500
```

or similar and then 

```python
number_of_asset_mappings = assert cursor.execute('SELECT COUNT(*) FROM location_asset_mappings').fetchone()[0] 
number_of_asset_mappings >= 1500
```